.github/actions/setup/action.yml

```diff
@@ -4,5 +4,8 @@ runs:
     - name: install required packages
       shell: bash
       run: |
+        sudo rm -rf /{usr/{local/{lib/{android,heroku},.ghcup,share/{dotnet,powershell,miniconda,swift}},share/{dotnet,miniconda,swift}},opt/{hostedtoolcache,microsoft},imagegeneration}
+        sudo docker system prune -a -f
+        sudo df -h
```

Comment:
Do we need this info? 

Otherwise looks good, we could merge if you give the go